shared/cfg/guest-hw.cfg

```diff
@@ -88,6 +94,12 @@ variants:
             drive_format=virtio-blk-device
             cd_format = scsi-cd
             scsi_hba = virtio-scsi-device
+        s390:
+            # Currently s390 only supports virtio-blk-ccw on s390
+            drive_format=virtio-blk-ccw
+            cd_format = scsi-cd
+            scsi_hba = virtio-scsi-ccw
```

Comment:
Please include the patch https://github.com/ldoktor/avocado-vt/commit/f5be0d87e152ff3d758dd7aa96bea0c096e0ecc6 before this commit as without it avocado-vt crashes when cdrom is used.